pkg/operation/botanist/health_check.go

```diff
@@ -661,7 +676,28 @@ func (b *Botanist) checkSystemComponents(
 	extensionConditions []ExtensionCondition,
 ) (*gardencorev1beta1.Condition, error) {
 
-	if exitCondition, err := checker.CheckSystemComponents(metav1.NamespaceSystem, condition, shootDeploymentLister, shootDaemonSetLister); err != nil || exitCondition != nil {
+	requirement, _ := labels.NewRequirement("type", selection.Equals, []string{"tunnel"})
+	tunnelSelector := labels.NewSelector().Add(*requirement)
+	podsList := &corev1.PodList{}
+	if err := b.K8sShootClient.Client().List(ctx, podsList, client.InNamespace(metav1.NamespaceSystem), client.MatchingLabelsSelector{Selector: tunnelSelector}); err != nil {
```

Comment:
I think, you can simplify this one:
```suggestion
	if err := b.K8sShootClient.Client().List(ctx, podsList, client.InNamespace(metav1.NamespaceSystem), client.MatchingLabels{"type": "tunnel"}; err != nil {
```

Reply:
okay